app/views/annotation_categories/_annotation_list.html.erb

```diff
@@ -6,10 +6,17 @@
     assignment_id: annotation_category.assignment.id,
     annotation_category_id: annotation_category.id
   ) %>
+  <%= javascript_tag nonce: true do %>
+    $(function() {
+      $('#add-annotation').click(function() {
+        add_annotation_prompt('<%= path %>')
```

Comment:
`e.preventDefault()`

And check to make sure we're doing this for all other cases too